Engine/Results/LiveTradingResultHandler.cs

```diff
@@ -1070,6 +1073,26 @@ protected virtual string CreateSafeChartName(string chartName)
         public void ProcessSynchronousEvents(bool forceProcess = false)
         {
             var time = DateTime.UtcNow;
+            var currentPortfolioValue = Algorithm.Portfolio.TotalPortfolioValue;
+
+            //// Sample the benchmark in real-time once we're certain that all feeds
+            //// and prices have been setup properly. In the past, we sampled at the beginning of the loop,
+            //// which caused issues with the alignment of data in the statistics calculation phase.
+            SampleBenchmark(time, Algorithm.Benchmark.Evaluate(time).SmartRounding());
+
+            if (time.Date != _previousProcessTime.Date)
+            {
+                // The benchmark will have an invalid first value since we don't
+                // calculate the percentage change between open and close for the first point.
+                //
+                // The performance series will contain the cumulative percent gain from market open to close
+                // for the first data point. This means that the first point between benchmark and performance
+                // are incompatible with each other, and must be removed for accurate calculations.
+                var portfolioValuePercentChange = _previousDailyPortfolioValue == 0 ? 0 : Math.Round((currentPortfolioValue - _previousDailyPortfolioValue) * 100 / _previousDailyPortfolioValue, 10);
+                SamplePerformance(time.Date, portfolioValuePercentChange);
```

Comment:
Were we not sampling performance for live trading before, seems like a bug? 🤔 

_Maybe_ some of this logic, private fields/calls (for example _previousDailyPortfolioValue/_previousProcessTime) could be moved to the shared `BaseResultsHandler` ?